pages/publish-nft-book/index.vue

```diff
@@ -82,18 +144,24 @@ const shouldShowActionButton = computed(() => {
   if (step.value === 0) {
     return hasFiles.value
   }
-  return true
+  return step.value < 3
 })
 
 const shouldDisableAction = computed(() => {
   if (step.value === 0) {
     return uploadFormRef.value?.uploadStatus !== ''
   } else if (step.value === 1) {
     return !registerISCN.value?.isFormValid
+  } else if (step.value === 2) {
+    return !mintNFT.value?.isFormValid
   }
   return false
 })
 
+const iscnQueryLink = computed(() => {
+  return `${APP_LIKE_CO_URL}/search?q=${wallet.value}`
```

Comment:
```suggestion
  return `${APP_LIKE_CO_URL}/search?owner=${wallet.value}`
```

Reply:
[✨ Add ISCN input handling and session resume functionality](https://github.com/likecoin/nft-book-press/pull/302/commits/dcd9ab26d297f0228caf4adc3b6eec8525b63792)